src/fread.c

```diff
@@ -96,12 +96,7 @@ typedef struct FieldParseContext {
 // Forward declarations
 static void Field(FieldParseContext *ctx);
 
-// note -- because ASSERT is doing literal char array concatenation, it is
-//   not possible to do translation of its messages without refactoring --
-//   essentially that would come down to creating f() g() in the code after
-//   macro expansion, which is not valid. These are internal errors, so just concede.
-#define ASSERT(cond, msg, ...) \
-  if (!(cond)) STOP(_("Internal error in line %d of fread.c, please report on data.table GitHub:  " msg), __LINE__, __VA_ARGS__) // # nocov
+#define ASSERT(cond, msg, ...) if (!(cond)) INTERNAL_STOPF(msg, __VA_ARGS__) // # nocov
```

Comment:
not sure we have any need for the `line %d` layer of complication; this PR allows to re-cast `ASSERT` as just being a wrapper of `INTERNAL_STOPF` that checks `cond`